index.html

```diff
@@ -6,7 +6,411 @@
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    48 часов
+                  </b>
+                </td>
+              </tr>
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <span class="slider-table-cell-title">
+                    Без подзарядки
+                  </span>
+                </td>
+              </tr>
+            </table>
+            </p>
+          </li>
+
+          <li class="slider-pagination-item">
+            <button class="slider-pagination-button" type="button">
+              <span class="visually-hidden">3 фото</span>
+            </button>
+            <p class="slider-description">
+              <b class="slider-description-title">
+                Делай селфи,<br>как Бен Стиллер!
+              </b>
+            </p>
+            <p class="slider-description">
+              Самая длинная палка для селфи доступна в нашем магазине. Восемь (Восемь, Карл!) метров длиной и весом всего
+              5 кг.
+            </p>
+            <p class="slider-paragraph">
+              <a class="slider-link" href="#">Подробнее</a>
+            </p>
+            <p class="slider-paragraph">
+            <table class="slider-table">
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    8,5 м
+                  </b>
+                </td>
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    5 кг
+                  </b>
+                </td>
+                <td class="slider-table-cell">
+                  <b class="slider-table-cell-info">
+                    Карбон
+                  </b>
+                </td>
+              </tr>
+              <tr class="slider-table-row">
+                <td class="slider-table-cell">
+                  <span class="slider-table-cell-title">
+                    Длина палки
+                  </span>
+                </td>
+                <td class="slider-table-item">
+                  <span class="slider-table-cell-title">
+                    Вес палки
+                  </span>
+                </td>
+                <td class="slider-table-item">
+                  <span class="slider-table-cell-title">
+                    Материал
+                  </span>
+                </td>
+              </tr>
+            </table>
+            </p>
+          </li>
+        </ol>
+      </section>
+
+      <section class="categories">
+        <h2 class="visually-hidden">Категории товаров</h2>
+        <ul class="categories-list">
+          <li class="categories-item">
+            <a class="categories-link" href="#"><img class src="" alt="Очки виртуальной реальности"></a><br>
+            <h3 class="categories-name">Виртуальная реальность</h3>
```

Comment:
Поскольку в стайлгайде для заголовка и картинки одни и те же состояние, значит они обёрнуты в одну ссылку. 
```suggestion
            <a class="categories-link" href="#">
              <img src="" alt="Очки виртуальной реальности">
              <h3 class="categories-name">Виртуальная реальность</h3>
            </a>        
```